spec/jobs/apply_api/import_applications_job_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+module ApplyApi
+  describe ImportApplicationsJob do
+    include ActiveJob::TestHelper
+
+    let(:application) { double("application") }
+
+    before do
+      allow(RetrieveApplications).to receive(:call).and_return([application])
```

Comment:
extreme nitpick: do we need the square brackets? I think this could just be `and_return(application)` ?

Reply:
At the moment, it's expected that `RetrieveApplications` returns an array of applications, which is then iterated over in `ImportApplicationsJob`.